### ISSUE TITLE
nan loss value for both training and validation. 

### ISSUE THREAD

**Issue by narminGhaffari** [clip-italian/clip-italian#24]
hi, I am receiving nan value for both train and valdtaion loss. My further investigation showed that 
`        text_outputs = self.text_model(
            input_ids=input_ids,
            attention_mask=attention_mask,
            token_type_ids=token_type_ids,
            position_ids=position_ids,
            deterministic=deterministic,
            output_attentions=output_attentions,
            output_hidden_states=output_hidden_states,
            return_dict=return_dict,
        )
`
the text_outputs is all nan. Do you have any suggestion what can be the problem?

**Comment by vinid** [comment]
Hello!

Probably hard to say without seeing the dataset and what the model is doing underneath.

Is this jax or pytorch? I'd try to give a shot at using pytorch

**Comment by narminGhaffari** [comment]
> Hello!
> 
> Probably hard to say without seeing the dataset and what the model is doing underneath.
> 
> Is this jax or pytorch? I'd try to give a shot at using pytorch

Here is how my dataset (json file) looks like: 

![image](https://github.com/clip-italian/clip-italian/assets/64002842/30553aba-df9a-4e0e-84f5-242044a3dd6b)

For the rest I am using everything from run_hybrid_clip.py script with all the parameters defined in .sh file.

**Comment by vinid** [comment]
Not sure, I'd use the pytorch loading pipeline or use https://github.com/mlfoundations/open_clip

**Comment by narminGhaffari** [comment]
![image](https://github.com/clip-italian/clip-italian/assets/64002842/57a3c547-a5d8-45b5-a4e6-7bcdc9521e74)
Is there any idea here that why image and text embedding both are returning NaN values? I checked the data_loaders and it seems that everything is working fine there.

**Comment by vinid** [comment]
Hello!

not sure what's happening without knowing the data and the process. You can try to see if it's a compatibility issue and use https://github.com/clip-italian/clip-italian/issues/20#issuecomment-1567913511 these requirements
